### ISSUE TITLE
Has anyone trained a tuned lens on Llama-3.1-8B or Llama-3.1-8B-Instruct?

### ISSUE THREAD

**Issue by joshbarua** [AlignmentResearch/tuned-lens#136]
Given these models are "optimized for multilingual dialogue use cases," I'm not sure if it makes sense to train the lenses using the same dataset used for Llama-3-8B.

**Comment by levmckinney** [comment]
We really have very little idea what is in any of the llama training sets after llama 1. You can use the same data we used for training the lens on llama 3 8b but I would avoid specifically using it for analysis of the models representations of different languages. You can always fall back to the Logit Lens and still use most of the libraries features.

I've been playing around with the idea of using some kind of self distillation to train the lenses and potentially get around this issue. For example you could generate a synthetic dataset of outputs from Llama-3.1-70B and you that to train the llama-8b lens. That would potentially remove the distribution shift but it would be quite expensive.

**Comment by joshbarua** [comment]
Makes sense, thank you! The self-distillation approach sounds interesting.
